packages/api/tests/api/users.test.ts

```diff
@@ -12,9 +12,82 @@ const sampleUser: Partial<User> = {
   githubId: '234234',
 };
 
+const sampleSignedInUser: Express.User = {
+  profile: {
+    id: 'aaa',
+  },
+  githubToken: 'abcd123',
+};
+
 const loggerSpy = jest.spyOn(logger, 'error').mockImplementation();
 
-describe('users API GET route', () => {
+describe('GET users/me', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks();
+  });
+
+  it('401 error when not logged in', async () => {
```

Comment:
These should always read as a sentence starting with `it`
```suggestion
  it('returns a 401 error when not logged in', async () => {
```